src/go/rdctl/pkg/shutdown/shutdown.go

```diff
@@ -57,34 +57,36 @@ func FinishShutdown(waitForShutdown bool, initiatingCommand InitiatingCommand) e
 	}
 	var err error
 	if err = directories.SetupLimaHome(); err != nil {
-		return err
-	}
-	limaCtlPath, err = directories.GetLimactlPath()
-	if err != nil {
-		return err
-	}
-	switch initiatingCommand {
-	case Shutdown:
-		err = s.waitForAppToDieOrKillIt(checkLima, stopLima, 15, 2, "lima")
-		if err != nil {
-			return err
-		}
-		// Check once more to see if lima is still running, and if so, run `limactl stop --force 0`
-		err = s.waitForAppToDieOrKillIt(checkLima, stopLimaWithForce, 1, 0, "lima")
-		if err != nil {
-			return err
-		}
-	case FactoryReset:
-		err = s.waitForAppToDieOrKillIt(checkLima, deleteLima, 15, 2, "lima")
+		logrus.Errorf("Ignoring error trying to get lima directory: %s", err)
+	} else {
+		limaCtlPath, err = directories.GetLimactlPath()
 		if err != nil {
-			return err
+			logrus.Errorf("Ignoring error trying to get path to limactl: %s", err)
+		} else {
+			switch initiatingCommand {
+			case Shutdown:
+				err = s.waitForAppToDieOrKillIt(checkLima, stopLima, 15, 2, "lima")
+				if err != nil {
+					return err
+				}
```

Comment:
Why do these errors abort the shutdown sequence? Should they also just be logged, and processing continues?